mitodl/mit-open
# Fix frontend sentry_env and release version

### What are the relevant tickets?
Followup for https://github.com/mitodl/hq/issues/5109

### Description (What does it do?)
This fixes two things:
- The bash script for getting verison was using `\d`, which caused errors on github actions and docker. Now uses `[0-9]`
- Fixed typo, `MITOPEN_ENV_RC -> MITOPEN_ENVIRONMENT_RC` (and _PROD)

### How can this be tested?
To some extent, need to release this. Here's what I did:

1. Run `./scripts/get_version/sh` in docker (it used to fail in docker)
2. View this github action... https://github.com/mitodl/mit-open/actions/runs/10291094957/job/28482598435 I made it to test the changes here. You can see that version is now picked up correctly.
